fitsio/fitslib.py

```diff
@@ -29,11 +29,14 @@
 from . import _fitsio_wrap
 from .util import FITSRuntimeWarning, cfitsio_version
 
+if sys.version_info >= (3, 0, 0):
```

Comment:
This was `sys.version_info > (3, 0, 0)` in another place in the code. IDK why. Just noting it here.